cumulus_etl/cli_utils.py

```diff
@@ -45,9 +47,14 @@ def add_aws(parser: argparse.ArgumentParser) -> None:
 def add_bulk_export(parser: argparse.ArgumentParser, *, as_subgroup: bool = True):
     if as_subgroup:
         parser = parser.add_argument_group("bulk export")
-    parser.add_argument("--since", help="start date for export from the FHIR server")
+    parser.add_argument(
+        "--since", metavar="TIMESTAMP", help="start date for export from the FHIR server"
+    )
     # "Until" is not an official part of the bulk FHIR API, but some custom servers support it
-    parser.add_argument("--until", help="end date for export from the FHIR server")
+    parser.add_argument(
+        "--until", metavar="TIMESTAMP", help="end date for export from the FHIR server"
+    )
+    parser.add_argument("--resume", metavar="URL", help="polling status URL from a previous export")
```

Comment:
Note that this flag is being added to both the standalone `cumulus-etl export` command and the default `cumulus-etl etl` command.